lib/pages/movie_info_screen/movie_details.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import '../../constants.dart';
+
+class MovieDetails extends StatelessWidget {
+  String? movieTitle;
+  String? movieReleaseDate;
+  num? movieVoteAverage;
```

Comment:
Should these be null?
Slightly (but not so) related to the comment above on `MovieDataModel`